rhizome/inference_endpoint/bin/download-lb-cert

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -ueo pipefail
+
+timestamp=$(date +%s)
+new_ssl_folder="/ie/workdir/ssl_$timestamp"
+rm -rf $new_ssl_folder
+rm -rf /ie/workdir/ssl_latest
+mkdir $new_ssl_folder
+
+if [ "$1" = "self-signed" ]; then
+    openssl req -x509 -out $new_ssl_folder/ubi_cert.pem -keyout $new_ssl_folder/ubi_key.pem -newkey rsa:2048 -nodes -sha256 -days 7 -subj '/CN=localhost'
+else
+    curl -f -L3 [FD00:0B1C:100D:5afe:CE::]:8080/load-balancer/cert.pem > $new_ssl_folder/ubi_cert.pem
```

Comment:
ah interesting, not sure how I memory-holed this, being on 8080. I can see why that's easier on the implementation, but I wonder if we should splice in an nftables rule to rewrite the port when dst=`[FD00:0B1C:100D:5afe:CE::].8080` to `80` to spare the customer the extra notation (cc @furkansahin ). Similar to the LB's own port rewriting feature for exactly the same reason (binding unprivileged ports, serving on a privileged port)

Reply:
Alright, that makes sense, I'll create a PR on that.

Reply:
https://github.com/ubicloud/ubicloud/pull/1993

Reply:
Made the switch to port 80 as the PR above was merged.